### ISSUE TITLE
Testing Library User Event Error

### ISSUE THREAD

**Issue by fadi-george** [vitest-dev/vitest#243]
### Describe the bug

Importing or re-exporting and then using user event from '@testing-library/user-event' leads to an error.

### Reproduction

Importing:
`import user from '@testing-library/user-event'`
or re-exporting

```
export { renderWithProviders as render, userEvent as user };
```

Then use it in a test:
```
user.click(someElement)
```

### System Info

```shell
System:
    OS: macOS 12.1
    CPU: (8) arm64 Apple M1 Pro
    Memory: 151.58 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.13.1 - ~/.nvm/versions/node/v16.13.1/bin/node
    npm: 8.1.2 - ~/.nvm/versions/node/v16.13.1/bin/npm
  Browsers:
    Brave Browser: 96.1.33.106
    Chrome: 96.0.4664.110
    Safari: 15.2
  npmPackages:
    @vitejs/plugin-react-refresh: ^1.3.6 => 1.3.6 
    vite: 2.7.4 => 2.7.4 
    vitest: 0.0.102 => 0.0.102
```

### Used Package Manager

npm

### Logs

```shell
TypeError: __vite_ssr_import_2__.user.click is not a function
```

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitest-dev/vitest/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitest-dev/vitest/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitest.dev/guide).
- [X] Check that there isn't [already an issue](https://github.com/vitest-dev/vitest/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitest-dev/vitest/discussions) or join our [Discord Chat Server](https://chat.vitest.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by wtchnm** [comment]
Got it working by adding @testing-library/user-event to [inline deps](https://vitest.dev/config/#deps-inline):
```
test: {
	deps: {
		inline: ['@testing-library/user-event']
	}
}
```

**Comment by antfu** [comment]
Caused by https://github.com/testing-library/user-event/issues/813

**Comment by sheremet-va** [comment]
Should be fixed in [0.0.120](https://github.com/vitest-dev/vitest/releases/tag/v0.0.120)
